README.md

```diff
@@ -1,2 +1,30 @@
 # hw01
 Repository for homework 01
+
+Hong Jin Jo
+====================
+
+Auto-biography
+---------------------
+
+This is a brief auto-biography of Hong Jin Jo.
+
+<img src="https://github.com/hongjinjo/myrepo/blob/master/JO_HONGJIN.jpg?raw=true">
+
+**Hong Jin Jo** was born in Seoul, Korea in 1988. He has attended the following schools:
+
+### Educational Background
+
+* Dongja Elementary School
+* Jayang Middle School
+* Korean Minjok Leadership Academy
+* Seoul National University
+
+He has acquired two sociology master degree from two institutions.
+
+1. Seoul National University
+2. The University of Chicago
```

Comment:
Just a quick suggestion. You don't need to put numbers in order. Markdown automatically understands the ordering. 
You could have written - 
```
1. Seoul National University
1. The University of Chicago
```

It would have converted to the last "1." to "2."